tests/conftest.py

```diff
@@ -84,6 +85,9 @@ def call_xfail(self, cmd):
         proc = self.unchecked_call(cmd)
         assert proc.returncode != 0
         assert proc.stderr != ""
+        def f(self2, reg):
```

Comment:
Maybe explain a bit what is going on here. At least I didn't find it obvious.
If I'm getting it right, how about: "Extend the Popen object with a match() method that runs a regex against the process sterr"

Reply:
Right! I've added a docstring explaining this.